pkg/skaffold/debug/transform_nodejs.go

```diff
@@ -96,8 +96,7 @@ func (t nodeTransformer) Apply(container *v1.Container, config imageConfiguratio
 			container.Args = rewriteNpmCommandLine(config.arguments, *spec)
 
 		default:
-			logrus.Warnf("Skipping %q as does not appear to invoke node", container.Name)
-			return nil
+			container.Env = rewriteNodeOptions(config.env, *spec)
```

Comment:
Just curious, at this point, was there already some hint that this container was node? Could I be a clown and have a java application with a `NODE_OPTIONS` ?

Reply:
you know what never mind, I read the description of the PR... :disappointed:

Reply:
It should be unusual (and discouraged) to have a mega-container with both a JVM and NodeJS application.  In the k8s sphere, they should be separate containers within the same pod.